test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_large_dim_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_size1_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_every_other_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_contig_vs_transposed_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_expand_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_index_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_non_contig_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_bool ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_int16 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_int32 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_int8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_bfloat16 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_bool ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_float32 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_float64 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_int16 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_int32 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_int64 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_int8 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_log1p_xpu_uint8 ... RuntimeError: "log1p_out" not implemented for 'ComplexFloat'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_1_xpu_int16 ... RuntimeError: mvlgamma is not implemented for Short
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_1_xpu_int32 ... RuntimeError: mvlgamma is not implemented for Int
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_1_xpu_int8 ... RuntimeError: mvlgamma is not implemented for Char
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_3_xpu_int16 ... RuntimeError: mvlgamma is not implemented for Short
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_3_xpu_int32 ... RuntimeError: mvlgamma is not implemented for Int
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_3_xpu_int8 ... RuntimeError: mvlgamma is not implemented for Char
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_5_xpu_int16 ... RuntimeError: mvlgamma is not implemented for Short
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_5_xpu_int32 ... RuntimeError: mvlgamma is not implemented for Int
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_5_xpu_int8 ... RuntimeError: mvlgamma is not implemented for Char
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_out_arg_all_dtypes_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_hard_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_hard_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_hard_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_hard_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_hard_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_lgamma_xpu_bfloat16 ... RuntimeError: "lgamma" not implemented for 'BFloat16'
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_lgamma_xpu_int64 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_lgamma_xpu_uint8 ... RuntimeError: at::isFloatingType(self.scalar_type())INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Gamma.cpp":72, please report a bug to PyTorch. Only support floating data type for now.
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_1_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_1_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_3_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_3_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_5_xpu_int64 ... RuntimeError: mvlgamma is not implemented for Long
test_unary_ufuncs.py::TestUnaryUfuncsXPU::test_reference_numerics_normal_mvlgamma_mvlgamma_p_5_xpu_uint8 ... RuntimeError: mvlgamma is not implemented for Byte
test_nn.py::TestNNDeviceTypeXPU::test_AdaptiveMaxPool3d_indices_xpu_float16 ... RuntimeError: "adaptive_max_pool3d_backward" not implemented for 'Half'
test_foreach.py::TestForeachXPU::test_unary_op_tensors_on_different_devices__foreach_abs_xpu_bool ... RuntimeError: "abs_cpu" not implemented for 'Bool'
test_torch.py::TestTorchDeviceTypeXPU::test_cauchy_kstest_xpu_bfloat16 ... RuntimeError: "cauchy_mkl_dpcpp" not implemented for 'BFloat16'
test_torch.py::TestTorchDeviceTypeXPU::test_cauchy_kstest_xpu_float16 ... RuntimeError: "cauchy_mkl_dpcpp" not implemented for 'Half'
test_torch.py::TestTorchDeviceTypeXPU::test_cauchy_no_inf_xpu_bfloat16 ... RuntimeError: "cauchy_mkl_dpcpp" not implemented for 'BFloat16'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_kstest_xpu_int16 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Short'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_kstest_xpu_int32 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Int'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_kstest_xpu_int64 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Long'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_kstest_xpu_int8 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Char'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_kstest_xpu_uint8 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Byte'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_xpu_int16 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Short'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_xpu_int32 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Int'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_xpu_int64 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Long'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_xpu_int8 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Char'
test_torch.py::TestTorchDeviceTypeXPU::test_geometric_xpu_uint8 ... RuntimeError: "geometric_scalar_dpcpp" not implemented for 'Byte'
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bfloat16_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_bool_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float32_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_float64_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int16_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int32_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int64_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_int8_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_bfloat16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_bool ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_float32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_float64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_int16 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_int32 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_int64 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_int8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_binary_ufuncs.py::TestBinaryUfuncsXPU::test_copysign_xpu_uint8_uint8 ... RuntimeError: iter.device(arg).type() == at::kXPUINTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Loops.h":411, please report a bug to PyTorch. argument 1: expected a XPU device but found cpu
test_reductions.py::TestReductionsXPU::test_empty_tensor_nonempty_slice_amax_xpu ... RuntimeError: iter.numel() > 0 && iter.ntensors() - iter.noutputs() == 1 && iter.noutputs() >= 1INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Reduce.h":1017, please report a bug to PyTorch. 
test_reductions.py::TestReductionsXPU::test_empty_tensor_nonempty_slice_amin_xpu ... RuntimeError: iter.numel() > 0 && iter.ntensors() - iter.noutputs() == 1 && iter.noutputs() >= 1INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Reduce.h":1017, please report a bug to PyTorch. 
test_reductions.py::TestReductionsXPU::test_prod_bool_xpu ... RuntimeError: "prod" not implemented for 'Bool'
test_reductions.py::TestReductionsXPU::test_tensor_compare_ops_empty_xpu ... RuntimeError: iter.numel() > 0 && iter.ntensors() - iter.noutputs() == 1 && iter.noutputs() >= 1INTERNAL ASSERT FAILED at "/home/gta/xunsongh/ipex-gpu/csrc/aten/operators/Reduce.h":1017, please report a bug to PyTorch. 
test_ops.py::TestMathBitsXPU::test_conj_view_cholesky_xpu_complex64 ... RuntimeError: "cholesky_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_cross_xpu_complex64 ... RuntimeError: "cross" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_dot_xpu_complex64 ... RuntimeError: "dot" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_gather_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_geqrf_xpu_complex64 ... RuntimeError: "geqrf_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_linalg_det_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_linalg_householder_product_xpu_complex64 ... RuntimeError: "orgqr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_linalg_qr_xpu_complex64 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_linalg_slogdet_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_lu_solve_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_ormqr_xpu_complex64 ... RuntimeError: "ormqr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_qr_xpu_complex64 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_scatter_add_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_scatter_xpu_complex64 ... RuntimeError: "Scatter" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_sigmoid_xpu_complex64 ... RuntimeError: "sigmoid_backward_out" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_solve_xpu_complex64 ... RuntimeError: "solve_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_symeig_xpu_complex64 ... RuntimeError: "symeig" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_take_along_dim_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_tanh_xpu_complex64 ... RuntimeError: "tanh_backward_out" not implemented for 'ComplexFloat'
test_ops.py::TestMathBitsXPU::test_conj_view_triangular_solve_xpu_complex64 ... RuntimeError: "triangular_solve_cpu" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_cholesky_xpu_complex64 ... RuntimeError: "cholesky_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_cross_xpu_complex64 ... RuntimeError: "cross" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_dot_xpu_complex64 ... RuntimeError: "dot" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_gather_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_geqrf_xpu_complex64 ... RuntimeError: "geqrf_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_linalg_det_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_linalg_householder_product_xpu_complex64 ... RuntimeError: "orgqr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_linalg_qr_xpu_complex64 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_linalg_slogdet_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_lu_solve_xpu_complex64 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_ormqr_xpu_complex64 ... RuntimeError: "ormqr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_qr_xpu_complex64 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_scatter_add_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_scatter_xpu_complex64 ... RuntimeError: "Scatter" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_sigmoid_xpu_complex64 ... RuntimeError: "sigmoid_backward_out" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_solve_xpu_complex64 ... RuntimeError: "solve_dpcpp" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_symeig_xpu_complex64 ... RuntimeError: "symeig" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_take_along_dim_xpu_complex64 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_tanh_xpu_complex64 ... RuntimeError: "tanh_backward_out" not implemented for 'ComplexFloat'
test_ops.py::TestCommonXPU::test_variant_consistency_eager_triangular_solve_xpu_complex64 ... RuntimeError: "triangular_solve_cpu" not implemented for 'ComplexFloat'
test_ops.py::TestGradientsXPU::test_fn_grad_cholesky_xpu_complex128 ... RuntimeError: "cholesky_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_cross_xpu_complex128 ... RuntimeError: "cross" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_dot_xpu_complex128 ... RuntimeError: "dot" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_gather_xpu_complex128 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_linalg_det_singular_xpu_complex128 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_linalg_det_xpu_complex128 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_linalg_householder_product_xpu_complex128 ... RuntimeError: "orgqr_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_linalg_qr_xpu_complex128 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_linalg_slogdet_xpu_complex128 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_lu_solve_xpu_complex128 ... RuntimeError: "lu_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_qr_xpu_complex128 ... RuntimeError: "qr_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_scatter_add_xpu_complex128 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_scatter_xpu_complex128 ... RuntimeError: "Scatter" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_sigmoid_xpu_complex128 ... RuntimeError: "sigmoid_backward_out" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_solve_xpu_complex128 ... RuntimeError: "solve_dpcpp" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_symeig_xpu_complex128 ... RuntimeError: "symeig" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_take_along_dim_xpu_complex128 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_tanh_xpu_complex128 ... RuntimeError: "tanh_backward_out" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_fn_grad_triangular_solve_xpu_complex128 ... RuntimeError: "triangular_solve_cpu" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_forward_mode_AD_scatter_add_xpu_complex128 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_forward_mode_AD_scatter_xpu_complex128 ... RuntimeError: "Scatter" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_forward_mode_AD_sigmoid_xpu_complex128 ... RuntimeError: "sigmoid_backward_out" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_forward_mode_AD_tanh_xpu_complex128 ... RuntimeError: "tanh_backward_out" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_grad_scatter_add_xpu_complex128 ... RuntimeError: "ScatterAdd" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_grad_scatter_xpu_complex128 ... RuntimeError: "Scatter" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_grad_sigmoid_xpu_complex128 ... RuntimeError: "sigmoid_backward_out" not implemented for 'ComplexDouble'
test_ops.py::TestGradientsXPU::test_inplace_grad_tanh_xpu_complex128 ... RuntimeError: "tanh_backward_out" not implemented for 'ComplexDouble'
